community/resources/docs-as-code.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: full-width
+group: community
+title: Docs as Code
+---
+
+To further expand documentation options for DevDocs, we offer a new opportunity for developers to contribute content through Community Engineering project repositories!
+
+Docs as Code is a new process that pulls content from public GitHub repositories and generates it as part of DevDocs. The DevDocs team helps you write add your content for the site. 
```

Comment:
"docs as code" on my understanding is a type of principles of handling the docs such as "one line - one sentence", re-use of reference links, review process, compiling (checks for syntax, fromatting, links, etc).
"Remote Doc Repo" is ambiguous because the devdocs repo is a remote doc repo in fact. But where we want to keeps docs is a codebase repo. The core distinction of the new approach is keeping docs __with__ the code.
So, "docs with code" or "multi-source docs" would be less ambiguous as to me.